zeebe/gateway-rest/src/test/java/io/camunda/zeebe/gateway/rest/controller/IncidentControllerTest.java

```diff
@@ -169,4 +193,195 @@ void shouldReturnIncidentProcessInstanceStatistics() {
+    when(incidentServices.searchIncidentProcessInstanceStatisticsByDefinition(
+            any(IncidentProcessInstanceStatisticsByDefinitionQuery.class)))
+        .thenReturn(INCIDENT_STATISTICS_ENTITY_SEARCH_QUERY_RESULT);
+
+    final var request =
+        """
+          {
+            "filter": { "errorHashCode": %d }
+          }
+        """
+            .formatted(errorHashCode);
+    // when/then
+    webClient
+        .post()
+        .uri(INCIDENT_STATISTICS_URL)
+        .accept(MediaType.APPLICATION_JSON)
+        .contentType(MediaType.APPLICATION_JSON)
+        .bodyValue(request)
+        .exchange()
+        .expectStatus()
+        .isOk()
+        .expectHeader()
+        .contentType(MediaType.APPLICATION_JSON)
+        .expectBody()
+        .json(
+            """
+            {
+              "items": [
+                {
+                  "processDefinitionId": "order-process-id",
+                  "processDefinitionKey": "2251799813685249",
+                  "processDefinitionName": "Order process",
+                  "processDefinitionVersion": 1,
+                  "tenantId": "<default>",
+                  "activeInstancesWithErrorCount": 3
+                }
+              ],
+              "page": {
+                "totalItems": 1,
+                "hasMoreTotalItems": false
+              }
+            }
+            """,
+            JsonCompareMode.STRICT);
+
+    final var result =
+        new IncidentProcessInstanceStatisticsByDefinitionQuery.Builder()
+            .filter(f -> f.errorMessageHashes(errorHashCode).states(IncidentState.ACTIVE.name()))
+            .build();
+    final ArgumentCaptor<IncidentProcessInstanceStatisticsByDefinitionQuery> captor =
+        ArgumentCaptor.forClass(IncidentProcessInstanceStatisticsByDefinitionQuery.class);
+    verify(incidentServices).searchIncidentProcessInstanceStatisticsByDefinition(captor.capture());
+    assertThat(captor.getValue()).isEqualTo(result);
+  }
+
+  @Test
+  void shouldRejectIncidentProcessInstanceStatisticsByDefinitionRequestWithoutBody() {
+    // given
+    final var expectedResponse =
+        String.format(
+            """
+                {
+                  "type": "about:blank",
+                  "title": "Bad Request",
+                  "status": 400,
+                  "detail": "Required request body is missing",
+                  "instance": "%s"
+                }""",
+            INCIDENT_STATISTICS_URL);
+    // when/then
+    webClient
+        .post()
+        .uri(INCIDENT_STATISTICS_URL)
+        .accept(MediaType.APPLICATION_JSON)
+        .contentType(MediaType.APPLICATION_JSON)
+        .exchange()
+        .expectStatus()
+        .isBadRequest()
+        .expectHeader()
+        .contentType(MediaType.APPLICATION_PROBLEM_JSON)
+        .expectBody()
+        .json(expectedResponse, JsonCompareMode.STRICT);
+  }
+
+  @Test
+  void shouldRejectIncidentProcessInstanceStatisticsByDefinitionRequestWithoutFilter() {
```

Comment:
🔧 We can also test the empty `filter` attribute in the request body that should fail.